crates/re_viewport/src/viewport.rs

```diff
@@ -262,6 +263,96 @@ impl<'a, 'b> egui_tiles::Behavior<SpaceViewId> for TabViewer<'a, 'b> {
         text
     }
 
+    #[allow(clippy::fn_params_excessive_bools)]
+    fn tab_ui(
+        &mut self,
+        tiles: &egui_tiles::Tiles<SpaceViewId>,
+        ui: &mut egui::Ui,
+        id: egui::Id,
+        tile_id: egui_tiles::TileId,
+        active: bool,
+        is_being_dragged: bool,
+    ) -> egui::Response {
+        // custom tab UI when we have a space view
+        let Some(egui_tiles::Tile::Pane(space_view_id)) = tiles.get(tile_id) else {
+            return egui_tiles::Behavior::<SpaceViewId>::tab_ui(
+                self,
+                tiles,
+                ui,
+                id,
+                tile_id,
+                active,
+                is_being_dragged,
+            );
+        };
+
+        // tab icon
+        let space_view = self
+            .space_views
+            .get(space_view_id)
+            .expect("Should have been populated beforehand");
```

Comment:
let's not crash here. We need to be super careful in the ui with any unwrap/expect calls and avoid them almost always in any viewer code.

This one for instance might easily happen with some frame delayed data - i.e. the space view got destroyed but the tabs didn't as just itself yet. Imho we should just silently ignore this case and render nothing (putting an `error_once` feels like the right thing, but then this may just spurious pop up with some users when it's probably not an issue after all)

Reply:
Guilty of copy/paste :) This applies the same to `fn pane_ui()` a few lines above, right? I'll change both.

Reply:
OK fixed. I cheated in `tab_ui()` and delegated to the status quo, which `debug_once!()`. Which btw makes me think that this situation must not occur that often otherwise we'd have seen that debug output.

Reply:
I don't think debug log is printed by default